src/toil/batchSystems/mesos/executor.py

```diff
@@ -94,6 +103,42 @@ def _sendStats(self, driver):
             log.debug("Sent stats message")
             sleep(30)
 
+    def _sendCurlRequest(self, endpoint, slaveID='', delta=''):
```

Comment:
This function is weirdly general and specific at the same time. The name sucks, too.